### PR TITLE
Issue #62 Enemy weapon cost display bug

### PR DESCRIPTION
**Pull request by bleehu**
fixed cost display but and weapon update bug

also used backslashes to break up really long lines for readability

### CODE REVIEW

**Comment by Turtlelord26** [comment]
What were the bugs?

**Comment by bleehu** [comment]
The cost of weapons here: https://54.200.202.155/monsterweapons only displays a single dollar sign.

**Comment by Turtlelord26** [comment]
I will need a couple points of confusion cleared up before I can pass judgment.

In enemies.py you change the 20th element of wepstring from 'suppression level' to 'fire select'.
1. Don't our c2.0 weapons have suppression levels? Why the deletion (and lack of usage elsewhere)?
Then, in lines 18-37 of enemy-weapons.py, I see only 19 elements from wepstring - it stops at cost and does not seem to include suppression level *or* fire select.
2. Reason for apparent exclusion or error?
3. The same thing happens in lines 45-48.

**Comment by bleehu** [comment]
For reasons nigh unexplainable, the wiring for suppression level is called fire select. 'cause that's a thing I did.
https://github.com/bleehu/CXDocs/blob/master/templates/monster_weapon_update.html#L113
I probably need to add that 20th element in enemy creation.

**Comment by bleehu** [comment]
Good catch. Made a bunch of updates.

File: enemy_weapons.py [hunk 1]
```diff
@@ -157,8 +176,23 @@ def validate_monster_weapon_map(form):
...
-    wepstring = (weapon['name'], weapon['damage'], weapon['mag'], weapon['type'], weapon['description'], weapon['author'], weapon['magCost'], weapon['r1'], weapon['r2'], weapon['r3'], weapon['acc1'], weapon['acc2'], weapon['acc3'], weapon['ap_level'], weapon['fire_rate'], weapon['refmod'], weapon['reload_dc'], weapon['move_speed_penalty'])
-    myCursor.execute("INSERT INTO monsters_weapons (name, damage, capacity, type, description, author, mag_cost, r1, r2, r3, acc1, acc2, acc3, ap_level, auto_fire_rate, reflex_modifier, reload_dc, move_speed_penalty) VALUES (E'%s', %s, %s, E'%s', E'%s', E'%s', %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s);" % wepstring)
+    wepstring = (weapon['name'], weapon['damage'], \
+        weapon['mag'], weapon['type'], weapon['description'], \
+        weapon['author'], weapon['magCost'], \
+        weapon['r1'], weapon['r2'], weapon['r3'], \
+        weapon['acc1'], weapon['acc2'], weapon['acc3'], \
+        weapon['ap_level'], weapon['fire_rate'], \
+        weapon['refmod'], weapon['reload_dc'], \
+        weapon['move_speed_penalty'], weapon['suppression_level'])
```
Comment (Turtlelord26) [inline enemy_weapons.py]: This instance of wepstring is missing weapon['cost'], though in enemies.py it does include the cost field.

File: enemy_weapons.py [hunk 2]
```diff
@@ -157,8 +176,23 @@ def validate_monster_weapon_map(form):
...
+    myCursor.execute("INSERT INTO monsters_weapons (name, \
+        damage, capacity, type, description, \
+        author, mag_cost, \
+        r1, r2, r3, acc1, acc2, acc3, \
+        ap_level, auto_fire_rate, \
+        reflex_modifier, reload_dc, \
+        move_speed_penalty, suppression_level) VALUES \
+        (E'%s', %s, %s, E'%s', E'%s', E'%s', %s, \
+        %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, E'%s');" % wepstring)
```
Comment (Turtlelord26) [inline enemy_weapons.py]: This block of stats is also missing the weapon cost.

**Pull request merged**